src/cascadia/TerminalSettingsEditor/MainPage.h

```diff
@@ -67,6 +67,8 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         void _Navigate(hstring clickedItemTag, BreadcrumbSubPage subPage);
         void _Navigate(const Editor::ProfileViewModel& profile, BreadcrumbSubPage subPage, const bool focusDeleteButton = false);
 
+        void _updateBackgroundForMica();
```

Comment:
```suggestion
        void _UpdateBackgroundForMica();
```
nit: all the other ones start capitalized